src/server/graphql/mutations/__tests__/removeTeamMember.test.js

```diff
@@ -0,0 +1,88 @@
+import MockDate from 'mockdate';
+import mockAuthToken from 'server/__tests__/setup/mockAuthToken';
+import {__now} from 'server/__tests__/setup/mockTimes';
+import removeTeamMember from 'server/graphql/mutations/removeTeamMember';
+import MockDB from 'server/__tests__/setup/MockDB';
+import MockPubSub from 'server/__mocks__/MockPubSub';
+import fetchAndSerialize from 'server/__tests__/utils/fetchAndSerialize';
+import getRethink from 'server/database/rethinkDriver';
+import {auth0ManagementClient} from 'server/utils/auth0Helpers';
+import DynamicSerializer from 'dynamic-serializer';
+import * as getPubSub from 'server/utils/getPubSub';
+import expectAsyncToThrow from 'server/__tests__/utils/expectAsyncToThrow';
+
+MockDate.set(__now);
+console.error = jest.fn();
+
+describe('removeTeamMember', () => {
+  test('promotes another member if the person removed was the lead', async () => {
+
```

Comment:
Are you still working on implementing this test case?

Reply:
yeah, i'll probably write that one from the remove-org-member mutation (uses the same removeAllTeamMembers helper function). this is just a reminder to add that one later, can remove if you like

Reply:
Ah I see. No need in removing it. I'd suggest you remove the `async () => {}`, though, which will make the test report to `jest` as "skipped", I think.